### PR TITLE
Event times

### PR DESCRIPTION
**Pull request by lexidelorey**

### CODE REVIEW

File: src/main/java/com/google/sps/data/Event.java [hunk 1]
```diff
@@ -19,6 +21,7 @@ public Event(EventBuilder builder) {
     this.eventId = builder.eventId;
     this.eventName = builder.eventName;
     this.dateTime = builder.dateTime;
+    this.startDateTime  = builder.startDateTime;
```
Comment (rglassett) [inline src/main/java/com/google/sps/data/Event.java:24]: What's the distinction between dateTime and startDateTime? Please add an implementation comment to clarify.

File: src/main/java/com/google/sps/servlets/EventServlet.java [hunk 2]
```diff
@@ -95,13 +96,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           (List<String>) entity.getProperty(Constants.RSVP_ATTENDEES_PARAM);
       String creator = 
           (String) entity.getProperty(Constants.CREATOR_PARAM);
-      Date endDate = (Date) entity.getProperty(Constants.END_DATE_PARAM);
+      Date startDateTime = (Date) entity.getProperty(Constants.START_TIME_PARAM);
+      Date endDateTime = (Date) entity.getProperty(Constants.END_DATE_PARAM);
```
Comment (rglassett) [inline src/main/java/com/google/sps/servlets/EventServlet.java]: The names `START_TIME_PARAM` and `END_DATE_PARAM` seem to disagree about whether these values are dates or times. Aim for consistency and clarity--in this case, it looks like we're dealing with dates.

File: src/main/webapp/eventScript.js [hunk 3]
```diff
@@ -278,8 +278,39 @@ function createEventNoResponse(event) {
...
+  topOfEvent.id = "top-event";
+
+  const creatorName = document.createElement('div');
+  creatorName.id = "event-creator";
+  fetch("/user")
+    .then(response => response.json())
+    .then(users => {
+      for (let i = 0; i < users.length; i++) {
+        if (users[i].id === event.creator) {
+          displayProfilePicture(users[i], document.getElementById("event-creator"), 'profile-pic-small');
```
Comment (rglassett) [inline src/main/webapp/eventScript.js]: You should be able to replace this call to `document.getElementById` by simply passing the `creatorName` element you instantiated above.

File: src/main/webapp/eventScript.js [hunk 4]
```diff
@@ -278,8 +278,39 @@ function createEventNoResponse(event) {
...
+  const topOfEvent = document.createElement('div');
+  topOfEvent.id = "top-event";
+
+  const creatorName = document.createElement('div');
+  creatorName.id = "event-creator";
+  fetch("/user")
+    .then(response => response.json())
+    .then(users => {
+      for (let i = 0; i < users.length; i++) {
+        if (users[i].id === event.creator) {
```
Comment (rglassett) [inline src/main/webapp/eventScript.js:290]: Consider [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) as an alternative to a `for` loop here.

**Review by rglassett — DISMISSED** [review]

**Review by ellisvivian — APPROVED** [review]

**Pull request merged**